### DISCUSSION TITLE
希望增加同步到obsidian的功能

### DISCUSSION THREAD

**Discussion by Felicityxi** [pagenote/pagenote#177] [Ideas]

**Comment by rowthan** [comment]
进展： https://github.com/coddingtonbear/obsidian-local-rest-api/pull/89
准备采用的 obsidian 插件，目前还有一个功能不具备同步的能力，已提交 PR，等待该插件准入后，PAGENOTE 将跟进 obsidian  同步能力。

**Reply by RocCore** [reply]
你好，请问现在如何备份自己的标记数据？如何实现云同步呢？现在我在浏览器的插件上都没找到有效的方法，只有一个插件数据备份。
